src/admin-events/admin-events.spec.js

```diff
@@ -0,0 +1,132 @@
+const ethers = require('ethers');
+const { getAddress } = require('@uma/contracts-node');
+
+// Load Config files
+const {
+  TransactionEvent,
+  FindingType,
+  FindingSeverity,
+  Finding,
+} = require('forta-agent');
+
+const config = require('../../agent-config.json');
+
+const chainId = 1;
+const votingAddressPromise = getAddress('Voting', chainId);
+
+const { handleTransaction } = require('./admin-events');
+
+/**
+ * TransactionEvent(type, network, transaction, receipt, traces, addresses, block)
+ */
+
+// Can this be imported from Forta Library?
```

Comment:
Note: Resolve the question in this comment, then delete the comment.

Reply:
Done